lightning/src/ln/peer_handler.rs

```diff
@@ -257,8 +257,13 @@ pub trait SocketDescriptor : cmp::Eq + hash::Hash + Clone {
 /// descriptor.
 #[derive(Clone)]
 pub struct PeerHandleError {
-	/// Used to indicate that we probably can't make any future connections to this peer, implying
-	/// we should go ahead and force-close any channels we have with it.
+	/// Used to indicate that we probably can't make any future connections to this peer (eg
+	/// because some of our/their required features were not implemented by them/us).
```

Comment:
suggestion: s/(eg because some of our/their required features were not implemented by them/us)/because one of us had required features that the other was missing